functions/AutoRemediateOrchestrator.js

```diff
@@ -2,17 +2,29 @@
 
 const CONFIG = require('./config')
 const AWS = require('aws-sdk')
+let delayedExecution;
+let delay;
+let queueURL;
+let type;
 
 module.exports.handler = (event, context, callback) => {
   console.log('Received event: ', JSON.stringify(event, null, 2))
-  console.log('Config settings: ', JSON.stringify(CONFIG, null, 2))
+  //console.log('Config settings: ', JSON.stringify(CONFIG, null, 2))
 
-  if (!event || !event.Records[0] || !event.Records[0].Sns || !event.Records[0].Sns.Message) {
+  if ((!event || !event.Records[0] || !event.Records[0].Sns || !event.Records[0].Sns.Message) && !event.Records[0].body) {
     callback(new Error('No event specified'))
     return
   }
-
-  let message = JSON.parse(event.Records[0].Sns.Message)
+  let message;
+  if(event.Records[0].body){
+    let sqsBody = JSON.parse(event.Records[0].body);
+    message = JSON.parse(sqsBody.Records[0].Sns.Message)
+    type=sqsBody.Records[0].Sns.Type
+  }
+  else{
+    message = JSON.parse(event.Records[0].Sns.Message)
+    type = event.Records[0].Sns.Type
+  }
   let AutoRemediate = 'AutoRemediate' + message.ruleId
```

Comment:
use `const` and backticks 
```suggestion
  const AutoRemediate = `AutoRemediate ${message.ruleId}`
```
ref: https://medium.com/better-programming/javascript-how-backticks-work-de269e0fb8ba